main.py

```diff
@@ -1,11 +1,36 @@
+class Car:
+    def __init__(self, WheelNumber, PSI, FuelType ):
+        self.WheelNumber = WheelNumber
+        self.PSI = PSI
+        self.FuelType = FuelType
 
-class Dog:
+        def get_WheelNumber(self):
+            return self.WheelNumber
 
-    def add_one(self, x):
-        return x + 1
-    def bark(self):
-        print("bark")
+        def get_PSI(self):
+            return self.PSI
 
-d = Dog()
-d.bark()
-print(d.add_one(5))
+        def get_FuelType(self):
+            return self.FuelType
+
+    def show(self):
+        print(self.WheelNumber, self.PSI, self.FuelType)
```

Comment:
this print doesn't make much sense, I think that you wanted to use something like 'format' and then it will be easier to write things around the arguments.
anyway I didn't define how to print it and you enlarged your head anyway, so no complains really.

Read about this:
https://www.programiz.com/python-programming/string-interpolation

Reply:
It's how it was shown in the video :D 
(I also didnt fully understand it)
Monkey see, Monkey do